README.md

```diff
@@ -1,12 +1,14 @@
 # customs-notification
 
-The objective of the this endpoint is as below: 
+The objective of this service is 
 
 1. Receive an update from CDS Backend System (Messaging) regarding a declaration that a CDS Client made earlier using Customs Declarations API
 
 2. Fetch client details(Client’s provided callback URL and Security Token) using the CDS Client ID received in header
 
-3. Notify the CDS Client with the given payload by calling notification gateway service with the payload
+3. Notify the CDS Client with the given payload by calling the notification gateway service with the payload or sending it to the pull queue
+
```

Comment:
implies notifications are deleted - how about `Provide two endpoints for the counting and deleting of retry blocks on notifications preventing them from being pushed`

Reply:
Fair point - tweaked text.